Podfile

```diff
@@ -1,3 +1,4 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
-pod 'React', :git => 'https://github.com/facebook/react-native'
+pod 'React'
```

Comment:
I’m not sure we also needed to change sourcing the React CocoaPod, but that’s okay for now. Pulling the podspec directly from the repo was an attempt to fix the problem that this PR _actually_ fixes. :+1: